packages/bun-plugin-svelte/test/fixtures/todo.svelte.ts

```diff
@@ -0,0 +1,13 @@
+export class Todo {
```

Comment:
Can you add a test that checks that this also works when this is using module.exports.Todo = instead?